pkg/app/app.go

```diff
@@ -258,7 +258,7 @@ func (c *App) AuthHandler(w http.ResponseWriter, r *http.Request) error {
 	if err := c.setOrUpdateSession(w, &s); err != nil {
 		return err
 	}
-	authURL := c.oauth2Helper.AuthCodeURL(state, oauth2.AccessTypeOffline)
+	authURL := c.oauth2Helper.OAuthConfig.AuthCodeURL(state, oauth2.AccessTypeOffline)
```

Comment:
I think we should care when `helper.OAuthConfig` is nil. Unless we have a check on OAuthConfig, we would face at unexpected behavior.